lbz/dev/server.py

```diff
@@ -56,9 +60,9 @@ def _get_route_params(self, org_path: str) -> Tuple[Union[str, None], Union[dict
                         acc += 1
                 if len(path) == acc:
                     return org_route, params
-        return None, None
+        raise ValueError
 
-    def _send_json(self, code: int, obj: dict, headers: dict = None) -> None:
+    def _send_json(self, code: int, obj: Union[str, dict], headers: dict = None) -> None:
```

Comment:
As I see, `obj` is only used at the end of this method, to be JSON dumped. But if it is already a string, that line will produce a different result - I am not sure if it is something that should be accepted, at least it doesn't look so:
```python
>>> json.dumps({"data": []}, indent=4, sort_keys=True)
'{\n    "data": []\n}'
>>> json.dumps('{"data": []}', indent=4, sort_keys=True)
'"{\\"data\\": []}"'
```